constants/constants.go

```diff
@@ -1,29 +1,62 @@
 package constants
 
+import "fmt"
+
 // OSType - Windows, Linux, MAC, Unknown
 type OSType string
 
+// ToolType - gonode, pasteld, dd-service, rq-service
+type ToolType string
+
+// ArchitectureType - amd64
+type ArchitectureType string
+
 const (
-	// Windows - Current OS is Windows
-	Windows OSType = "Windows"
-	// Linux - Current OS is Linux
-	Linux OSType = "Linux"
-	// Mac - Current OS is MAC
-	Mac OSType = "MAC"
-	// Unknown - Current OS is unknown
-	Unknown OSType = "Unknown"
+	// DownloadBaseURL - The base URL of the pastel release files
+	DownloadBaseURL string = "https://download.pastel.network"
 
 	// PastelConfName - pastel config file name
 	PastelConfName string = "pastel.conf"
 
 	// PastelUtilityConfigFilePath - The path of the config of pastel-utility
 	PastelUtilityConfigFilePath string = "./pastel-utility.conf"
 
+	// PastelUtilityLogFilePath - The path of the log of pastel-utility
+	PastelUtilityLogFilePath string = "./pastel-utility-remote-log.txt"
+
 	// PipRequirmentsFileName - pip install requirements file name
 	PipRequirmentsFileName string = "requirements.txt"
 
 	// DupeDetectionImageFingerPrintDataBase - dupe_detection_image_fingerprint_database file
 	DupeDetectionImageFingerPrintDataBase string = "dupe_detection_image_fingerprint_database.sqlite"
+
+	// PastelUtilityDownloadURL - The path of pastel-utility for install supernode remote
+	PastelUtilityDownloadURL string = "https://github.com/pastelnetwork/pastel-utility/releases/download/v0.5.8/pastel-utility-linux-amd64"
+
+	// RequirementDownloadURL - The path of requirement.txt for install pip
+	RequirementDownloadURL string = "https://download.pastel.network/machine-learning/requirements.txt"
+)
+
+// Windows type
+const (
+	Windows OSType = "Windows"
+	Linux   OSType = "Linux"
+	Mac     OSType = "MAC"
+	Unknown OSType = "Unknown"
+)
+
+// Tool type
+const (
+	WalletNode ToolType = "gonode/walletnode"
+	SuperNode  ToolType = "gonode/supernode"
+	PastelD    ToolType = "pasteld/pastel"
+	DDService  ToolType = "dd-service/ddservice"
+	RQService  ToolType = "rq-service/rqservice"
+)
+
+// Architecture type
+const (
+	AMD64 ArchitectureType = "amd64"
```

Comment:
define only one const block

Reply:
I thought that this field could be added in the future.

Reply:
there are 3 const blocks in this file.
the comment is regarding that. there should be one block because that is what blocks are for.